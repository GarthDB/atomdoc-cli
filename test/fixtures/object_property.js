const objectWithFunction = {
  /**
   *  Public: object function
   *
   *  * `node` {Object} Spidermonkey AST node
   *  * `arr` {Array} of {Objects}
   *  * `returnsArr` {Array} of Returns
   */
  objectFunction(value) {
    return value;
  }
}
