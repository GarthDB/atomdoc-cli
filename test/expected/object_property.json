[
  {
    "originalText": "Public: object function\n\n* `node` {Object} Spidermonkey AST node\n* `arr` {Array} of {Objects}\n* `returnsArr` {Array} of Returns",
    "visibility": "Public",
    "description": "object function",
    "summary": "object function",
    "arguments": [
      {
        "name": "node",
        "description": "{Object} Spidermonkey AST node",
        "type": "Object",
        "isOptional": false
      },
      {
        "name": "arr",
        "description": "{Array} of {Objects}",
        "type": "Array",
        "isOptional": false
      },
      {
        "name": "returnsArr",
        "description": "{Array} of Returns",
        "type": "Array",
        "isOptional": false
      }
    ],
    "start": 31,
    "end": 196,
    "className": false,
    "name": "objectFunction",
    "definitionLine": 9
  }
]
